packages/cli/src/commands/build.ts

```diff
@@ -608,6 +651,59 @@ async function doBuild(
   );
 }
 
+export async function determineFrameworkVersion(
+  frameworkRecord: UnboxPromise<ReturnType<typeof detectFrameworkRecord>>,
```

Comment:
```suggestion
  frameworkRecord: Framework,
```

Can we simplify the type here?

Reply:
Updated in https://github.com/vercel/vercel/pull/9009#:~:text=Resolve%20conversation-,PR%20feedback,-fbf562e